crates/turborepo-lockfiles/src/npm.rs

```diff
@@ -135,6 +138,16 @@ impl Lockfile for NpmLockfile {
     fn encode(&self) -> Result<Vec<u8>, crate::Error> {
         Ok(serde_json::to_vec_pretty(&self)?)
     }
+
+    fn global_change_key(&self) -> Vec<u8> {
```

Comment:
Is there a commit on your local machine that changes this function?

Reply:
Oops, had that stashed